packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -1488,9 +1514,21 @@ export class DATE extends AbstractDataType<AcceptedDate> {
   }
 
   toBindableValue(date: AcceptedDate) {
+    if (this.options.plain) {
+      return this._applyTimezone(date).format('YYYY-MM-DD HH:mm:ss.SSS');
```

Comment:
I don't think we should use applyTimezone for plain dates

Reply:
Should plain dates work with `globalTimeZoneConfig`?
This which would essentially allow the user what timezone to store the plain dates in otherwise it would default to UTC.

Reply:
After further review and testing, I have removed this.